### PR TITLE
Docs:Update Developer Guide

### PR DESCRIPTION
**Pull request by Pughal77**
Update changes to import implementation to the developer guide

Update sequence diagram to include `CsvUtil`

Add manual testing for importing of persons

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -978,3 +1003,105 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `import i|file.csv` (file has a row with missing compulsory value)<br>
+      Expected: All valid rows are added to the address book. Error message shown in the error report for invalid rows.
+
+   1. Test case: `import i|file.csv` (file has extra headers)<br>
+       Expected: Only the compulsory headers are read. Optional headers are read if present. Extra headers are ignored.
+
+   1. Test case: `import i|file.txt` (file is empty CSV file)<br>
+      Expected: Error message shown in the error report. No change in the address book.
+
+   1. Test case: `import i|file.csv` (file has rows whose number of values is unequal to the number of headers)<br>
```
Comment (Zer0Legion) [inline docs/DeveloperGuide.md:1106]: should this be all 1s

File: docs/diagrams/ImportSequenceDiagram.puml [hunk 2]
```diff
@@ -10,6 +10,7 @@ participant ":ArgumentTokenizer" as ArgumentTokenizer LOGIC_COLOR
 participant "multi:ArgumentMultimap" as ArgumentMultimap LOGIC_COLOR
 participant ":ParserUtil" as ParserUtil LOGIC_COLOR
 participant "i:ImportCommand" as ImportCommand LOGIC_COLOR
+participant "<<class>>CsvUtil" as CsvUtil LOGIC_COLOR
```
Comment (Zer0Legion) [inline docs/diagrams/ImportSequenceDiagram.puml:13]: should it be <<class>>\nCsvUtil instead?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -369,12 +369,21 @@ The activity diagram is as follows:
 
 ### Import contacts from CSV file
 
+The `import` command allows users to import contacts from a CSV file. Users can specify the file path of the CSV file to 
+import contacts from and with the validation and checking of the CSV rows, person objects can be added to the addressbook.
+
 #### Implementation
```
Comment (jayllo-c) [inline docs/DeveloperGuide.md:375]: Seems well-detailed

**Review by jayllo-c — COMMENTED** [review]
Other than the error @Zer0Legion has mentioned, the DG was updated diligently!

**Pull request closed without merging**